pkg/reconciler/managed/reconciler.go

```diff
@@ -637,7 +640,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	// Be wary of adding additional complexity.
 
 	log := r.log.WithValues("request", req)
-	log.Debug("Reconciling")
+	log.Debug("Reconciling if not paused")
```

Comment:
I think for debug logging purposes it's safe to just leave this as it was. It will be quickly followed by a notice that reconcile is paused if that is the case.